core-backend/common/src/lib.rs

```diff
@@ -393,3 +398,42 @@ pub trait BackendTermination<E: BackendExt, M: Sized>: Sized {
         (ext, memory, termination_reason)
     }
 }
+
+pub use log;
```

Comment:
`pub use log` and `pub const PTR_SPECIAL` should be moved above
and as well isn't it better to introduce the code of the macros while solving appropriate issue? It's not used anywhere here

Reply:
This macroses will used in sandbox as well in future

Reply:
I see, I'm talking about removal of currently unused code, cause it could be changed in future once used

Reply:
I don't see any unused code here =| what are you talking about?

Reply:
I have moved up log and PTR_SPECIAL